flink-examples/flink-examples-streaming/src/main/scala/org/apache/flink/streaming/scala/examples/windowing/WindowWordCount.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.streaming.scala.examples.windowing
+
+import org.apache.flink.api.java.utils.ParameterTool
+import org.apache.flink.api.scala._
+import org.apache.flink.examples.java.wordcount.util.WordCountData
+import org.apache.flink.streaming.api.scala.StreamExecutionEnvironment
+
+/**
+  * Implements a windowed version of the streaming "WordCount" program.
+  *
+  * <p>
+  * The input is a plain text file with lines separated by newline characters.
+  *
+  * <p>
+  * Usage: <code>WordCount
+  * --input &lt;path&gt;
+  * --output &lt;path&gt;
+  * --window &lt;n&gt;
+  * --slide &lt;n&gt;
+  * </code><br>
+  * If no parameters are provided, the program is run with default data from
+  * {@link org.apache.flink.examples.java.wordcount.util.WordCountData}.
```

Comment:
Probably better to have a Scala-style link here i.e. `[[org.apache.flink.examples.java.wordcount.util.WordCountData]]`